docs/app-directory/overview.md

```diff
@@ -1,24 +1,171 @@
+
+### DNS/SRV Records
+
+Another approach to support AppD service discovery (resolution) is through use of existing domain name service (DNS) implementations that are broadly used on the Internet today (see: [RFCs](https://www.isc.org/community/rfcs/dns/)). Name service implementations can be considered critical infrastructure and are proven stable with over twenty years of use. Name services can be used both through public Internet or locally deployed intranet, which provides optionality to deployment schemes.
+
+More specifically, resolution of an AppD service instance (host location) can be implemented using DNS "service records" (SRV) providing the host instance, protocol and associated port. The following is a well known description of a SRV record ([RFC2782](https://tools.ietf.org/html/rfc2782)):
+
+```
+zone name { _service._proto.name. TTL  class  SRV priority weight port target.}
+```
+
+- *service*: the symbolic name of the desired service.  For AppD service, this must be identified as "**_appd**"
+- *proto*: the transport protocol of the desired service; this is usually either [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) or [UDP](https://en.wikipedia.org/wiki/User_Datagram_Protocol).  For AppD service **_tcp** must be used.
+- *name*: the domain name for which this record is valid, ending in a dot.  For AppD service,  the name should directly map to the application identifier domain.
+- *TTL*: standard DNS [time to live](https://en.wikipedia.org/wiki/Time_to_live) field.
+- *class*: standard DNS class field (this is always *IN*).
+- *priority*: the priority of the target host, lower value means more preferred.
+- *weight*: A relative weight for records with the same priority, higher value means more preferred.
+- *port*: the TCP or UDP port on which the service is to be found. For AppD service, TCP should always be used. 
+- *target*: the canonical hostname of the machine providing the service, ending in a dot. This would be the host where the AppD service is running. 
+
+For AppD Service the SRV record MUST use the following definitions:
+
+- service = **_appd**
+- proto = **_tcp**
+- name = must map to the domain of the application identifier . Example:  the **name** for application identifier **"app1@appd.foo.com"** would be **"appd.foo.com"**
+
+*AppD service through DNS / SRV records:*
+
+![img](assets/appd_dns.png)
+
+#### Known domains
+
+Although SRV records provide the means of resolving the location of an AppD service for a specific domain, there could be a need to know what domains exist in the universe. This would be a list of domains representing all known directory instances. It is recommended that  the FDC3/FINOS organization publish a list of known domains which support AppD services. This publication can be handled in multiple ways, such as structured files or API endpoints. This proposal shall not provide a qualified solution to achieve this, but rather draw attention to a potential requirement.
```

Comment:
```suggestion
Although SRV records provide the means of resolving the location of an app directory service for a specific domain, there could be a need to know what domains exist in the universe. This would be a list of domains representing all known directory instances. It is recommended that the FDC3/FINOS organization publish a list of known domains which support AppD services. This publication can be handled in multiple ways, such as structured files or API endpoints. This proposal shall not provide a qualified solution to achieve this, but rather draw attention to a potential requirement.
```